app/src/org/commcare/android/database/connect/models/PersonalIdCredential.kt

```diff
@@ -0,0 +1,112 @@
+package org.commcare.android.database.connect.models
+
+import org.commcare.android.model.PersonalIdValidAndCorruptCredential
+import org.commcare.android.storage.framework.Persisted
+import org.commcare.models.framework.Persisting
+import org.commcare.modern.database.Table
+import org.commcare.modern.models.MetaField
+import org.javarosa.core.services.Logger
+import org.json.JSONArray
+import org.json.JSONException
+import org.json.JSONObject
+import java.io.Serializable
+
+@Table(PersonalIdCredential.STORAGE_KEY)
+class PersonalIdCredential : Persisted(), Serializable {
+
+    @Persisting(1)
+    @MetaField(META_UUID)
+    var uuid: String = ""
+
+    @Persisting(2)
+    @MetaField(META_APP_ID)
+    var appId: String = ""
+
+    @Persisting(3)
+    @MetaField(META_OPP_ID)
+    var oppId: String = ""
+
+    @Persisting(4)
+    @MetaField(META_ISSUED_DATE)
+    var issuedDate: String = ""
+
+    @Persisting(5)
+    @MetaField(META_TITLE)
+    var title: String = ""
+
+    @Persisting(6)
+    @MetaField(META_ISSUER)
+    var issuer: String = ""
+
+    @Persisting(7)
+    @MetaField(META_LEVEL)
+    var level: String = ""
+
+    @Persisting(8)
+    @MetaField(META_TYPE)
+    var type: String = ""
+
+    @Persisting(9)
+    @MetaField(META_ISSUER_ENVIRONMENT)
+    var issuerEnvironment: String = ""
+
+    @Persisting(10)
+    @MetaField(META_SLUG)
+    var slug: String = ""
```

Comment:
adding any new fields to the DB needs a DB migration, current the app crashes on this branch on updating from an older version.